src/counter.rs

```diff
@@ -16,19 +16,23 @@ impl Counter {
         }
     }
 
-    pub fn add(&mut self, key: &str) {
+    pub fn counts(self) -> AHashMap<String, u64> {
```

Comment:
I think it'd be better to solve the merging of `Counter`s inside the `Counter` type, by adding `merge(&mut self, other: &Counter)` instead of this method.

Reply:
Done